CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## 0.0.3
+## v 0.0.3 ?
```

Comment:
On `pytest-inmanta` we put the date between bracket (https://raw.githubusercontent.com/inmanta/pytest-inmanta/master/CHANGELOG.md). Let's do that here as well for consistency.
```suggestion
## v 0.0.3 (?)
```